src/Microsoft.OData.Edm/Csdl/CsdlJsonReaderSettings.cs

```diff
@@ -4,7 +4,7 @@
 // </copyright>
 //---------------------------------------------------------------------
 
-#if NETSTANDARD2_0
+#if NETCOREAPP

```

Comment:
Confirm it this compiler flag applies to net8.0 even after they drop "Core" from the framework name

Reply:
It applies to all .NET Core and .NET 5+ versions. See: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives